src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -2,15 +2,25 @@ import { useVideos } from '@/api/hooks'
 import { MessageDialog, StudioContainer, TransactionDialog, VideoPreviewPublisher } from '@/components'
 import { absoluteRoutes } from '@/config/routes'
 import { useAuthorizedUser, useDeleteVideo, useDrafts, useEditVideoSheet, useSnackbar } from '@/hooks'
-import { Grid, Pagination, Tabs, Text } from '@/shared/components'
-
+import { Grid, Pagination, Tabs, Text, Select } from '@/shared/components'
 import React, { useEffect, useState } from 'react'
 import { useNavigate } from 'react-router-dom'
 import { EmptyVideos, EmptyVideosView } from './EmptyVideosView'
-import { PaginationContainer, StyledDismissibleMessage, TabsContainer, ViewContainer } from './MyVideos.styles'
+import {
+  PaginationContainer,
+  SortContainer,
+  StyledDismissibleMessage,
+  TabsContainer,
+  ViewContainer,
+} from './MyVideos.styles'
 import { removeVideoFromCache } from '@/utils/cachingAssets'
+import { VideoOrderByInput } from '@/api/queries'
 
 const TABS = ['All Videos', 'Published', 'Drafts', 'Unlisted'] as const
+const SORT_OPTIONS = [
+  { name: 'ascending', value: VideoOrderByInput.CreatedAtAsc },
```

Comment:
I think we should change naming here to "Newest/Oldest"

Reply:
let's make it `Newest first`/`Oldest first`